jupyterhub/jupyterhub/overlays/jupyterhub-idle-culler/deploymentconfig.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: jupyterhub-idle-culler
+spec:
+  selector:
+    app: jupyterhub-idle-culler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: jupyterhub-idle-culler
+    spec:
+      containers:
+        - name: jupyterhub-idle-culler
+          image: quay.io/odh-jupyterhub/jupyterhub-img:v0.3.2
```

Comment:
I set img version to 0.3.2 in anticipation of a future update which will include the change on the jupyterhub-odh side